docs/discovery-handlers/onvif.md

```diff
@@ -156,6 +361,65 @@ helm install akri akri-helm-charts/akri \
     --set onvif.configuration.discoveryDetails.discoveryTimeoutSeconds=2
 ```
 
+### Accessing Secret data in Broker
+
+The Onvif a sample broker (`akri-onvif-video-broker`) can be configured to access Secret and ConfigMap data, if configured, it expects the Secret and ConfigMap data are
```

Comment:
nit: remove `a` ... `The Onvif sample broker...`

Reply:
fix in https://github.com/project-akri/akri-docs/pull/89